### ISSUE TITLE
Not all Cargo airport stand is used

### ISSUE THREAD

**Issue by praveenjabakar** [CitiesSkylinesMods/TMPE#1354]
When using vanilla , say if i have 3 Cargo airport stand, all placed as row  then cargo planes seems to coming to all 3. but with our Mod, it seems always goes to first Cargo stand. I am not sure because of this path finding. Is it possible to switch off using path finding for airport ?

**Comment by krzychu124** [comment]
The thing is the aircraft must want to go to different stand. That decision is done outside of TMPE. I'll take a look later.

**Comment by praveenjabakar** [comment]
Again last night , i have ran with vanilla. It goes to all airport stand and noticed, some airplanes are circling around if no taxi way. 

Morning i have switched on, cleared traffic. all goes to 1 stand + noticed, airplanes are not circling instead it just lands and vehicles overlapes to each other, means 2 or 3 planes in 1 location standing,

I badly need this mod for main junction , lane connector, vehicle trestriction , dynamic lane , speed limit and timed traffic light. Please help to fix for air craft or ignore it to vanilla for now is highly appreciated.

Thanks a lot brother.

**Comment by krzychu124** [comment]
TM:PE is only searching for path for the aircrafts for now, because TM:PE is replacing whole pathfinding. Pathfinding for aircrafts is 1:1 as vanilla, no tweaks or any changes. Other behaviour like `stands assignment`, `taxiing`, `taking off`, `landings` are fully handled by vanilla code. Because of that none of TM:PE tools work for taxiways - we don't support such vehicles at all and I don't know yet why they do what they do with TMPE enabled 🤷‍♂️  

Adding support will be quite hard as all code is designed for road/rail vehicles only and Aircrafts when moving can do both (move on the ground and also fly, obviously and transition between states (take off/landing)) so a lot of additional code for state detection (on ground/flying) will be required. 

It will be huge task.

**Comment by praveenjabakar** [comment]
Thanks bro. Is there a way to enable full log?

**Comment by praveenjabakar** [comment]
Looks like problem with CitiesHarmony used. I just replaced CitiesHarmony new version ( yeah may be it is under beta not sure) and recompiled. Now Cargo stand is going like expected + all my garbage collector is collecting too. I dont what is the purpose of CitiresHarmony dll. Anyway let me try to check.

**Comment by aubergine10** [comment]
Which version of harmony were you using previously?

**Comment by praveenjabakar** [comment]
Currently used in nuget and this mod uses below
  <package id="CitiesHarmony.API" version="2.0.0" targetFramework="net35" />
  <package id="CitiesHarmony.Harmony" version="2.0.4" targetFramework="net35" developmentDependency="true" />

I have used the pre-release vesion as below

  <package id="CitiesHarmony.API" version="**2.1.0-rc**" targetFramework="net35" />
  <package id="CitiesHarmony.Harmony" version="**2.2.0-rc**" targetFramework="net35" developmentDependency="true" />

**Comment by aubergine10** [comment]
I'm assuming you're on Linux?

**Comment by praveenjabakar** [comment]
Windows 10 brother

**Comment by krzychu124** [comment]
You can use stable version of Harmony, I also tested experimental on 2 days ago, both work without any issues. TMPE is shipped with CitiesHarmony,Api.dll which in game communicates and requests the main Cities.Harmony.dll, either [this (stable)](https://steamcommunity.com/workshop/filedetails/?id=2040656402) or [that (experimental)](https://steamcommunity.com/sharedfiles/filedetails/?id=2399204842) depending on which is installed. 
Experimental is currently at final phase so will be released as Stable soon :tm: AFAIK

Without Cities Harmony mod TM:PE won't work at all and it's required since 11.6.x+. Up to v11.5.2 previous version was included(embedded) but now migrated to external version to be easier to keep updated. Also you can't have multiple versions of Harmony installed, just a side note.

**Comment by praveenjabakar** [comment]
Agreed bro,let me wait for stable release then and check it again. Currently TM PE mod screws up cargo whereas vanilla pathfinding works fine.

![image](https://user-images.githubusercontent.com/66250112/152641385-73eabe09-c3c8-4105-a459-5314e2ffe9fd.png)

**Comment by krzychu124** [comment]
I can't reproduce that problem... What I'm doing wrong? (city is super tiny, <1k)🤔 

![image](https://user-images.githubusercontent.com/19638970/152659950-6a9c50c3-11c9-4266-9a7c-25a252e82374.png)
![image](https://user-images.githubusercontent.com/19638970/152659964-1609b8bb-b025-4014-bd14-601368920a91.png)
![image](https://user-images.githubusercontent.com/19638970/152660028-90ff318a-32ba-4c76-99b4-308937f8fbb5.png)
